rclpy/rclpy/qos.py

```diff
@@ -20,8 +20,43 @@
 from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
 
 
+class QoSPolicyKind(IntEnum):
+    """
+    Enum for types of QoS policies that a Publisher or Subscription can set.
+
+    This enum matches the one defined in rmw/incompatible_qos_events_statuses.h
+    """
+
+    RMW_QOS_POLICY_INVALID = 1 << 0
```

Comment:
why the RMW prefix? Considering the class name, I would just left `INVALID`. In that way, it can be used like: `QoSPolicyKind.INVALID`, which is really readable IMO.

Reply:
Done.